grails-app/conf/UrlMappings.groovy

```diff
@@ -1,10 +1,9 @@
-import grails.util.Environment
-
 class UrlMappings {
 
     static mappings = {
 
         "/proxy"(controller: "proxy", action: "index")
+        "/sitemap/**"(controller: "sitemap", action: "index")
 
```

Comment:
Thats needed as we want to get to the index action yet we want the URL asking for sitemap.xml